dashpay/src/main/kotlin/org/dashevo/dashpay/BlockchainIdentity.kt

```diff
@@ -541,9 +544,9 @@ class BlockchainIdentity {
         for (unregisteredUsername in usernames) {
             val salt = saltForUsername(unregisteredUsername, true)
             val fullUsername = if (unregisteredUsername.contains(".")) {
-                unregisteredUsername
+                unregisteredUsername.toLowerCase()
```

Comment:
this fixes a bug that prevented a user from register a username with capital letters